test/controllers/MrnEntryControllerSpec.scala

```diff
@@ -61,26 +61,48 @@ class MrnEntryControllerSpec extends ControllerSpecBase {
     when(secureMessagingConfig.isSecureMessagingEnabled).thenReturn(true)
   }
 
+  val validUrl = "https://www.google.com"
+
   "MrnEntryController on onPageLoad" should {
 
     "return Ok (200) response" when {
 
       "cache is empty" in {
 
         val controller = mrnEntryController()
-        val result = controller.onPageLoad(fakeRequest)
+        val result = controller.onPageLoad()(fakeRequest)
 
         status(result) mustBe OK
       }
 
       "cache contains MRN data" in {
 
         val controller = mrnEntryController(validAnswers.copy(mrn = MRN(mrn)))
-        val result = controller.onPageLoad(fakeRequest)
+        val result = controller.onPageLoad()(fakeRequest)
 
         status(result) mustBe OK
       }
     }
+
+    "not update the mrnPageRefererUrl value in the cash" when {
```

Comment:
Nit: Must admit that English is not my first language, but should it not be "cache" ?

Reply:
Lol, must have money on my mind! :laughing: